cookiecutter.json

```diff
@@ -0,0 +1,8 @@
+{
+  "project_name": "Website",
+  "project_slug": "{{ cookiecutter.project_name.lower().replace(' ', '_').replace('-', '_') }}",
```

Comment:
@jpmartinspt Why "-" to "_" we have a lot of repos with "-"?

Reply:
mistake! updated now..